frontend/packages/operator-lifecycle-manager/src/components/topology/operator-link-tab-section.tsx

```diff
@@ -1,12 +1,18 @@
 import * as React from 'react';
 import { GraphElement } from '@patternfly/react-topology';
 import { ManagedByOperatorLink } from '@console/internal/components/utils/managed-by';
+import { NodeType } from '@console/knative-plugin/src/topology/topology-types';
 import TopologySideBarTabSection from '@console/topology/src/components/side-bar/TopologySideBarTabSection';
 import { TYPE_WORKLOAD } from '@console/topology/src/const';
 import { getResource } from '@console/topology/src/utils';
 
 export const getManagedByOperatorLinkSideBarTabSection = (element: GraphElement) => {
-  if (element.getType() !== TYPE_WORKLOAD) return undefined;
+  if (
+    element.getType() !== TYPE_WORKLOAD &&
+    element.getType() !== NodeType.Revision &&
+    element.getType() !== NodeType.KnService
```

Comment:
We need a better way to resolve this. probably check on data `isOperatorBacked`?

Reply:
Using `element.getData().data?.isKnativeResource`